micro-rdk/src/common/conn/viam.rs

```diff
@@ -859,22 +926,32 @@ where
                 None
             });
 
-        self.executor.spawn(async move {
-            #[allow(unused_mut)]
-            let mut srv = GrpcServer::new(robot, GrpcBody::new());
-            #[cfg(feature = "local-signaling")]
-            if let Some(ss) = ss {
-                srv.register_signaling_server(ss);
+        log::info!("spawning task for new HTTP2 connection");
+        self.executor.spawn(
+            async move {
```

Comment:
I'm not sure why `cargo fmt` decided to push this down, but it did. Perhaps it is related to the `inspect` calls at the end?

Anyway, the only interesting changes in here are new logs.